src/main/java/main/MainClass.java

```diff
@@ -0,0 +1,30 @@
+package main;
+
+import model.DosageForm;
+import model.Drug;
+import model.Order;
+import userInterface.Bucket;
+import userInterface.User;
+
+public class MainClass {
+    public static void main(String[] args) {
+        Pharmacy pharmacy = new Pharmacy();
+        Drug drug = new Drug("A", "AA", 1.1, true, DosageForm.HARD_CAPSULE, "AAA");
+        pharmacy.addDrugToBase(drug);
+
+        Drug foundDrug = pharmacy.findDrugById(1);
+        pharmacy.addDrugToBucket(foundDrug);
```

Comment:
try 
`pharmacy.addDrugToBucket(foundDrug1);`
`pharmacy.addDrugToBucket(foundDrug2);`
is it still working?